build/lib/src/state/filesystem_cache.dart

```diff
@@ -209,12 +208,14 @@ class InMemoryFilesystemCache implements FilesystemCache {
     Encoding encoding = utf8,
     required Uint8List Function() ifAbsent,
   }) {
+    // Encodings other than utf8 do not use `_stringContentCache`. Read as
+    // bytes then convert, instead.
     if (encoding != utf8) {
       final bytes = readAsBytes(id, ifAbsent: ifAbsent);
       return encoding.decode(bytes);
     }
 
-    // Check _stringContentCache first to use it as a cache for conversion of
+    // Check `_stringContentCache` first to use it as a cache for conversion of
```

Comment:
I can't seem to put a comment at the right place, but line 223-235 could basically be replaced by a call like `final bytes = readAsBytes(id, ifAbsent: ifAbsent);` (as above).

Reply:
So it can ... thanks! Done.